gio/src/list_store.rs

```diff
@@ -15,6 +18,15 @@ pub trait ListStoreExtManual {
 
     #[doc(alias = "g_list_store_sort")]
     fn sort<F: FnMut(&Object, &Object) -> Ordering>(&self, compare_func: F);
+
+    #[cfg(any(feature = "v2_64", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v2_64")))]
+    #[doc(alias = "g_list_store_find_with_equal_func")]
+    fn find_with_equal_func<P: IsA<glib::Object>, F: Fn(&Object, &Object) -> glib::ffi::gboolean>(
```

Comment:
Shouldn't this be:

```suggestion
    fn find_with_equal_func<P: IsA<glib::Object>, F: Fn(&IsA<Object>, &IsA<Object>) -> glib::ffi::gboolean>(
```


?

Reply:
It also should be returning a normal bool and not a glib:ffi::gboolean